### DISCUSSION TITLE
Anytype collaboration

### DISCUSSION THREAD

**Discussion by xaviibaez** [anyproto/contributors#65] [Q&A] [answered]
Hello!, this is Xavi.

First of all thanks for the help  Andrew gave me about setting up my environment to start collaborating with you.

I'm starting to read all the issues listed in anytype-ts on github. How can I assign myself on a particular issue? 

Also, Is there any slack group of collaborators where I can join?
Thanks.

**Comment by ra3orblade** [comment, marked as answer]
We do not have group in Slack especially for contributors, but we have a group in Slack for community testers, but you can be added there if you want. Second thing is that issues on Github are pretty new, so if you want to check list of issues the best way is to browse community forums at https://community.anytype.io/. Our internal bug-tracker is unfortunately private, but it is connected to this forums and we work with community on bug reports and feature requests.

**Comment by angeloreale** [comment]
Hi @ra3orblade , can we draft a collaboration guideline document and make it more accessible (README / direct link on [website](https://anytype.io/contributors))? Also, can I get added to the Slack? 

I'm willing to consider contributing to Anytype both from a personal standpoint, but possibly as an internal ideation platform for [@dreampipcom/zeus](https://github.com/dreampipcom/zeus).

There would be some crucial features we'd need for that to be valuable for us though:
- Algebraic Formulas (e.g. that can trasverse relationships);
- CSV/JSON imports/exports (e.g. that can remove duplicates and also trigger automated actions);
- Charts (e.g. that can plot at least 2 Y axis over an X axis (e.g. number fields on a table over time);
- Repeated actions (e.g. add template entry daily)
- Automated triggers (e.g. add entry on a different table with reference to this when a new [this] page is created);
- Integrations/Webhooks (e.g. send a request with a custom body payload when entry is added/modified, listen to requests on a server and add/modify entries on a specific page/table).

Obviously, we'd not expect you to do anything other than your usual business, thus considering a more comprehensive collaboration, which I couldn't find the guidelines for and would like to be able to communicate about.

I've joined the forum already, and can start writing RFCs, but I feel there needs to be a better understanding from my part first.

Thank you for this tool,
Angelo.

**Reply by angeloreale** [reply]
Adding to list: autoincrement / UID field

**Reply by angeloreale** [reply]
Grid view needs quick drag to fill (copy value over rows), or rather a Table view.

**Reply by angeloreale** [reply]
Copy-pasting / exporting Views (Grid, List) to Markdown should include rows

**Comment by angeloreale** [comment]
For anyone coming late, here it is: https://github.com/anyproto/.github/blob/main/docs/CONTRIBUTING.md
